openedx/openedx-platform
# feat: adds sortable fields to studio content search index [FC-0059]

## Description

This PR adds new datetime fields to the serialized library block metadata: `created`, `modified`, and `last_published`, and adds support for sorting search results by these dates or the block title/`display_name`. 

Also adds `last_published` to the list of fields we can filter by, so that e.g. sorting by "Recently Published" can also filter out unpublished content.

This change should not impact any current users, as  content libraries v2 are not yet ready for production use.

## Supporting information

Part of https://github.com/openedx/frontend-app-course-authoring/issues/1038

## Testing instructions

See https://github.com/openedx/frontend-app-course-authoring/pull/1147 for full testing instructions.

## Deadline

ASAP -- to meet project deadlines

## Other information

These changes can be deployed by running the management command:

```
python manage.py cms reindex_studio --experimental
```
